corehq/apps/translations/models.py

```diff
@@ -92,3 +95,26 @@ def get_translations(cls, lang, key=None, one=False):
                 return dict([(key, val[0]) for key, val in translations.items()])
             else:
                 return translations
+
+
+class TransifexOrganization(models.Model):
+    slug = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
+    api_token = models.CharField(max_length=255)
```

Comment:
Checked that even django user api token is in sql table and i think we do the same for repeaters so thought it was safe to keep this here as well. Happy to move this back to localsettings if this is not the secure way.

Reply:
This is probably fine since they are API tokens which are easily revoked but you could also encrypt them like this: https://github.com/dimagi/commcare-hq/blob/02f8c09a4584f6b96f178ebaa1022234b707fc42/corehq/motech/repeaters/views/repeaters.py#L139

Reply:
Encrypt using django secret. Thats nice.
Adding that.